gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
Please undo. I guess Android Studio keeps updating this for you. You can choose "Do not remind me for this project" when it asks you the next time.

Reply:
I am sorry for this repeated mistake.
Actually build fails for my project with the current build version and certain other changes that I have made here https://github.com/commons-app/apps-android-commons/pull/2289. Hence I need to revert build.gradle to debug it in phone

Reply:
I have modified the code in my next commit

Reply:
Am sorry to hear that. It might be fruitful to make #2289 merge ready and put it in master. For starters, I am merging #2275.

Reply:
:+1: